sale_section_and_note_revamp/tests/common.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo.addons.sale.tests.common import TestSaleCommon
+
+
+class TestDisplayLineMixinCommon(TestSaleCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+
+        SaleOrder = cls.env["sale.order"]
+        SaleOrderLine = cls.env["sale.order.line"]
+
+        # create a generic Sale Order with all classical products and empty pricelist
+        cls.sale_order = SaleOrder.create(
+            {
+                "partner_id": cls.partner_a.id,
+                "partner_invoice_id": cls.partner_a.id,
+                "partner_shipping_id": cls.partner_a.id,
+                "pricelist_id": cls.company_data["default_pricelist"].id,
+            }
+        )
+
+        cls.sol_section_1 = SaleOrderLine.create(
+            {
+                "sequence": 1,
```

Comment:
Has odoo will popluate all lines with sequence=10, wouldn't it be better to not force the sequence at creation and change it only if required in tests?